alignment/FairAlignmentHandler.cxx

```diff
@@ -106,14 +102,9 @@ void FairAlignmentHandler::RecomputePhysicalAssmbBbox() const
 {
     TObjArray* pPhysNodesArr = gGeoManager->GetListOfPhysicalNodes();
 
-    TGeoPhysicalNode* pPhysNode = nullptr;
-    TGeoShapeAssembly* pShapeAsb = nullptr;
-
-    Int_t iNbNodes = pPhysNodesArr->GetEntriesFast();
-    for (Int_t iInd = 0; iInd < iNbNodes; ++iInd) {
-        pPhysNode = dynamic_cast<TGeoPhysicalNode*>(pPhysNodesArr->At(iInd));
+    for (auto pPhysNode : TRangeDynCast<TGeoPhysicalNode>(pPhysNodesArr)) {
```

Comment:
Looks good. Did not know this.

Reply:
Same for me, nice to learn about this functionality, it may be useful later (even we use less ROOT style containers/collection with direct access recently).